src/About/Repl/Repl/Host/Window.js

```diff
@@ -4,80 +4,73 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import {html, thunk, forward, Effects, Task} from 'reflex';
-import {merge, batch, tag, tagged} from "../../../../Common/Prelude";
-import {ok, error} from "../../../../Common/Result";
+import {html, thunk, forward, Effects, Task} from 'reflex'
+import {merge, batch, tag, tagged} from '../../../../Common/Prelude'
+import {ok, error} from '../../../../Common/Result'
 
+import type {ID, EvaluationResult} from '../Host'
+import type {Never} from 'reflex'
 
-import type {ID, EvaluationResult} from "../Host"
-import type {Never} from "reflex"
-
-
-const DELETE = new String('delete');
+const DELETE = 'delete'
```

Comment:
I'll change it back, but then i'll leave this file in the `standard-flow` ignore list, as it complains:
```
standard-flow: Use JavaScript Standard Style (with Flow types) (https://github.com/gozala/standard-flow)
  /Users/mathieu/mozilla/browserhtml/src/About/Repl/Repl/Host/Window.js:14:16: Do not use String as a constructor.
  /Users/mathieu/mozilla/browserhtml/src/About/Repl/Repl/Host/Window.js:69:57: eval can be harmful.```